src/serviceWorker.ts

```diff
@@ -12,139 +12,136 @@
 
```

Comment:
Just to double check, nothing in this file was explicitly changed other than formatting, correct?  It doesn't look like it, but I figured I'd ask to double check, it looks like prettier was just run on it so that changed spacing/indentation.

Reply:
Yeah, I just ran prettier.